src/components/AddMovie/AddMovie.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import './AddMovie.css';
+
+const AddMovie = () => (
+    <button className="addMovieBtn">+ ADD MOVIE</button>
+);
+
+export default AddMovie
```

Comment:
It is better to have newline at the end of files

Reply:
will fix